src/playercards/customizable/UpgradeSheetLibrary.ttslua

```diff
@@ -60,6 +63,22 @@ function onLoad(savedData)
   selfId = getSelfId()
 
   maybeLoadLivingInkSkills()
+  xmlTable = {
```

Comment:
```suggestion
  local xmlTable = {
```

Reply:
making it local there does not allow it to be used by createRowCheckboxes and createRowTextfield. Should I make it local at the top of the script?

Reply:
Ah sorry, you're right. It's fine as is imo